ilastik/applets/dataSelection/dataSelectionGui.py

```diff
@@ -415,11 +415,17 @@ def addFiles(self, roleIndex, startingLaneNum=None):
 
     def addFileNames(self, paths: List[Path], startingLaneNum: int, roleIndex: int):
         # If the user didn't cancel
-        for path in paths or []:
+        # we iterate through every path and add it. now bulk addition.
```

Comment:
I think this comment is unnecessary because it just reiterates what is already happening in code.